.github/workflows/build-and-deploy.yml

```diff
@@ -154,6 +170,16 @@ jobs:
         run: |
           cp -avr wiki-tutorials/target/generated-docs/ target/generated-docs/website/pages/learning/
 
+      - name: Create output for journeys
+        run: |
+          chmod +x journeys/scripts/createOutput.sh
+          journeys/scripts/createOutput.sh
+          
+      - name: Copy output of journeys
+        run: |
+          cp -avr journeys/target/generated-docs/ target/generated-docs/website/pages/journeys/
+          mv target/generated-docs/website/pages/journeys/journeys target/generated-docs/website/pages/journeys/journey
```

Comment:
Why is that needed? It looks like createOutput is doing something in that direction already.